libbpf-rs/src/program.rs

```diff
@@ -73,6 +73,8 @@ pub enum ProgramType {
     StructOps,
     Ext,
     Lsm,
+    SkLookup,
```

Comment:
I do not know why this code was pulled into my PR.  I did not edit anything in program.rs.

Will this be a problem?

Reply:
yeah, you have 3 extra commits. You need to pull latest master, rebase your commit on top of it and force-push your branch.